### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by baxik88**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)\
assign-2

## What did you learn from the changes you have made?
code

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
any

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
no

## How were these changes tested?
manually

## A reference to a related issue in your repository (if applicable)
-

## Checklist
- [x ] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by x-rojas-io — APPROVED** [review]
Hi!

Thank you for submitting your DL assignments.

Your A2 and Labs 4-6 are now Complete.

Comments:
A2, minor issues in Task 4

- You had a Dropout(0.2) layer in your model but left it commented out. Adding it would reduce overfitting and stabilize training.
- Change trainable=False to trainable=True to allow the model to fine-tune the embeddings. This will likely improve performance.
- Increase the number of epochs (e.g., 20-30 epochs).
- Use early stopping (EarlyStopping(monitor='val_loss', patience=5)).
- Lower the learning rate of the optimizer to stabilize the training.

Thanks,
Nestor [LS]

**Comment by baxik88** [comment]
Hi Nestor!

Thanks for your fast feedback.

Take care,
Igor
